frontend/src/javascript/utils/parsingUtils.js

```diff
@@ -117,5 +117,5 @@ function parseCircles(value) {
 }
 
 function parseArrayOrString(value) {
-    return value.includes(" ") ? parseArray(value) : value;
+    return value.includes(" ") || value.includes("\n") ? parseArray(value) : value;
```

Comment:
Wiesz co, myk polegał na tym, że przykładem tablicy jest też string. I tutaj to rozróżnienie to jest takie prowizoryczne rozwiązanie, żeby można było również wyświetlać ciągi znaków w tablicy bez tam jakieś konwersji stringa na vector (tak wiem, że jak string będzie miał w środku spację to się błędnie wyświetli). Ogólnie to w głowie mam zadanie do zrobienia na później, żeby dodać różne tryby parsowania tych obiektów. Np. w grafie, żeby można było wybrać czy graf ma być ważony czy nie ważony (lista trójek czy par), czy wyróżnianie wierzchołków to jest lista wierzchołków do wyróżnienia czy tablica bitów czy dany wierzchołek ma być wyróżniony czy nie itp. 

Prośba o sprawdzenie czy to parsowanie działa poprawnie i dla tablic i dla stringów.

Reply:
Wszystkie istniejące projekty nadal działają, to sprawdzałem.
Obecne rozwiązanie już było dziurawe, jak sam napisałeś, zakładało że string nie ma spacji. Teraz dodałem dodatkowe założenie, że nie ma nowej linii. Zgodzę się, że to nie jest idealne rozwiązanie, ale w obrębie tego taska raczej nic więcej nie zrobię. Rozdzielanie elementów tablicy nowymi liniami zamiast spacjami znacznie ułatwia uogólnienie konwertera na vector.

Sprawdziłem stringi:
```
string a = "abcdef";
string b = "abc def";
string c = "abc\ndef";
string d = "ab cd\nef";
```
I rezultat, nie jest idealnie, ale o ile to prosty string to działa:
![obraz](https://user-images.githubusercontent.com/27860636/233864488-4c52e92c-3bd7-4e80-a113-40676d2f27a2.png)